src/Pages/Volunteer.jsx

```diff
@@ -448,6 +463,156 @@ export const Volunteer = () => {
                   writing.
                 </FormHelperText>
               </FormGroup>
+              <Typography
+                variant="body1"
+                sx={{ fontWeight: "bold", pt: 2 }}
+              >
+                Volunteer Hold Harmless Agreement:
+              </Typography>
+              <FormHelperText>
+                Agreement made this {currentDateAsOrdinal} day of {currentMonth}, {currentYear} by and between A Refuge for Saving the Wildlife, Incorporated&copy; (hereinafter referred to as RSW&copy;), located in Northbrook, IL, and (hereinafter referred to as Volunteer).
+              </FormHelperText>
+              <FormHelperText>
+                Whereas RSW&copy; desires to accept the gratuitous services of the Volunteer, and the Volunteer desires to provide gratuitous services to RSW&copy;.
+              </FormHelperText>
+              <FormHelperText>
+                In consideration of the following premises and mutual covenants, and other good and valuable considerations, the receipt and sufficiency of which are hereby acknowledged, it is mutually covenanted and agreed by and between the parties hereto as follows:
+              </FormHelperText>
+              <FormGroup
+                sx={{ pl: 4 }}
+              >
+                <Controller 
+                  control={control}
+                  name="wishes_to_be_volunteer"
+                  rules={{ validate: (value) => value === "Yes" || "Response is required" }}
```

Comment:
I'm thinking for the ones that are basically just checkboxes that have to be "yes", we should change the error message to "Agreement is required". Do you think that sounds clearer?

Reply:
I do think that sounds clearer. I'll make that change!